Dockerfile

```diff
@@ -0,0 +1,27 @@
+# Create Python Image
+FROM python:3.8.4-buster as Base
+
+# Install Poetry
+RUN curl -sSL https://raw.githubusercontent.com/python-poetry/poetry/master/get-poetry.py | python
+
+ENV PATH="${PATH}:/root/.poetry/bin"
+
+# Create Workdir & Copy the Code
+WORKDIR /DevOps-Course-Starter
+COPY . /DevOps-Course-Starter
+
+# Expose the Port
+EXPOSE 5000
+
+FROM base as development
+
+
+RUN poetry install
+ENTRYPOINT [ "poetry", "run", "flask", "run", "--port", "5000" , "--host", "0.0.0.0"]
+
+FROM base as production
+
+ENV FLASK_ENV=production
+RUN poetry install
```

Comment:
You've separated this command out from the base image, but it's identical to the one you use for development. That might suggest we have them both in base, but here it would be better to instead install with the `--no-dev` argument, to reduce the need to install dependencies that aren't used in production

Reply:
Included the argument on poetry install and tested. Works fine with the argument included